react-app/src/store/reviews.js

```diff
@@ -114,7 +114,11 @@ const reviewReducer = (state = initialState, action) => {
     let newState;
     switch (action.type) {
         case GET_REVIEWS:
+            // console.log('review store', action.reviews)
+            // console.log('state', state)
             newState = Object.assign({}, state);
+            // newState.allReviews = {};
+            console.log('all reviews', newState.allReviews)
```

Comment:
one last thing before I approve it all, can you comment out this new console.log